lib/ansible/plugins/action/win_reboot.py

```diff
@@ -1,16 +1,15 @@
-# (c) 2016, Matt Davis <mdavis@ansible.com>
+# (c) 2018, Matt Davis <mdavis@ansible.com>
```

Comment:
Why change year? Why not 2016-2018 then?

Reply:
I just figured copyright dates were supposed to be updated to the current year. I don't pretend to understand the law around the dates.

Reply:
I'm not sure. Normally, the year in the year of content creation (first publication) and 2016 means "copyright protected 2016 and forward".
If there's content created during a period of time it'd be a range or a list of years.

https://smallbusiness.chron.com/year-copyright-statement-mean-62027.html
https://www.copyrightlaws.com/copyright-notice-year/

Reply:
Also, there was some guideline where the next updated would add a separate line with themselves, but won't edit other lines.

cc @abadger